app/uk/gov/hmrc/helptosavefrontend/views/helpinformation/after_4_years.scala.html

```diff
@@ -18,10 +18,17 @@
 
 @(maybeDate:Option[LocalDate])(implicit messages: play.api.i18n.Messages)
 
+@getNextMonth(date: LocalDate) = @{
+date.getMonthValue + 1 match {
```

Comment:
BAD! There should be better ways of doing that.  You should be able to add one month to a LocalDate and then get the month

Reply:
Thanks for catching this Dave, I completely forgot that plusMonths() existed. I am making the changes now.